src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -271,4 +277,20 @@ public JSONObject getTeamActivity(String accessToken) {
         }
     }
 
+    public  static  Boolean  getUserSubscription() {
```

Comment:
Why do we have double spaces between these words?

Reply:
Not quite sure how that came. Removing